sashimi-webapp/test/unit/specs/database/stringManipulator.spec.js

```diff
@@ -0,0 +1,72 @@
+import StringManipulation from 'src/database/stringManipulation';
```

Comment:
String manipulator looks like a useful helpers function can all of use can benefit from. You may consider putting it in `src/helpers` next time.